### ISSUE TITLE
Extensions deactivating after a couple days 

### ISSUE THREAD

**Issue by swandi** [ThemeFuse/Unyson#1605]
Hi, 

I've seen this post issue recurring throughout some years but nobody seems to have tapped on what's going on here. I've built my homepage layout with Unyson, but after a couple days it disappears. I haven't been able to track down exactly when it happens as I'm not actively working on the site, but whenever I go back to make a change or ask the client to look at something, the homepage is gone. I click on "Unyson" in the sidebar and nothing is active, I need to re-"download" Page Builder, and then it all appears again.

I've tried several suggestions in my searching but have lost track of all the darts I've thrown. I'm also not a PHP expert. 

Unyson was updated 3 days ago. I am using the Furnde theme.

Is there maybe a plugin just for Page Builder without the entire Unyson framework? Or is there a way I can just copy the HTML to not rely on Unyson until this is resolved? Otherwise, help?

**Comment by danyj** [comment]
some years? bud , Unyson is out since Dec last year , and in each post with this issue ( about 8 ) none of the users provided any quality lead or test ground for Unyson team to test and find the issue. Please provide as much server and plugins info as you can than all might be able to find out what this is.  Also provide your theme fles , there might be something inside that Unyson theme devs are not doing correctly.

**Comment by swandi** [comment]
Ok fair enough, late nights and confusing debug threads have obviously
confused me.

How am I supposed to provide you all this information? You want me to
migrate my entire site with all its settings to a testing ground, and
where? Should I make an admin setting for you to be able to log in to the
site I'm working on? Understand that most people installing layout page
builders are doing so because they are not tech professionals. I created a
github account just to post here. I get that it's difficult with so many
unknowns. Please be more specific in how I can help.

**Comment by moldcraft** [comment]
@swandi The most important thing is `error.log`, please find it and provide it here or ask your hosting to send it to you. There we can find some error messages that can be very helpful.

**Comment by swandi** [comment]
Thanks @moldcraft , I don't have one of those files made already so I have turned on debugging mode and added a line of code to create a debug file. I'll monitor the site waiting for the error to happen again and keep you updated!

**Comment by moldcraft** [comment]
> waiting for the error to happen

There can be no error, it can be a errorless bug in code. But I wanted to take a look at error log maybe there is one. Ok, thank you for monitoring it

**Comment by swandi** [comment]
OK this problem has finally affronted itself again. I would get these two lines of code: 

`[23-Jun-2016 13:12:31 UTC] PHP Warning:  session_start(): Cannot send session cache limiter - headers already sent (output started at /home/nathaneli/public_html/wp-content/themes/furnde-child/functions.php:12) in /home/nathaneli/public_html/wp-content/plugins/unyson/framework/helpers/class-fw-flash-messages.php on line 241
[23-Jun-2016 13:12:31 UTC] PHP Warning:  Cannot modify header information - headers already sent by (output started at /home/nathaneli/public_html/wp-content/themes/furnde-child/functions.php:12) in /home/nathaneli/public_html/wp-includes/pluggable.php on line 1167`

I'm not sure what it means. My child-theme function file doesn't even have 12 lines: 

`<?php
add_action( 'wp_enqueue_scripts', 'theme_enqueue_styles' );
function theme_enqueue_styles() {
    wp_enqueue_style( 'parent-style', get_template_directory_uri() . '/style.css' );

}

?>`

Any clues?

**Comment by moldcraft** [comment]
Remove `?>` at the end of the file

**Comment by swandi** [comment]
Thanks, that seems to work. Time will tell if in another 27 days the
plugins may deactivate again...

Deanna Dionne

Creative Contractor
web - print - product

DeannaDionne.com <http://deannadionne.com/>

c: 586-489-0957

**Comment by swandi** [comment]
Hi. I have noticed again today the plugin deactivated. It seems to happen after no action or prompt-- The last change/login I made to the site was June 28, and everything goes along fine until today it's all just deactivated and I need to click "download" on page builder for my site content to appear again. 

There is no recent entry in the debug log. I did what you said 14 years ago with removing the end of the functions.php file. I have no idea how to provide more information that may help because it seems completely random to me.
